src/templates/pages/blog/data.json

```diff
@@ -0,0 +1,301 @@
+                "name": "Our blog"
+            },
+            {
+                "link": "#",
+                "name": "The Basilisks NFT"
+            }
+        ]
+    },
+    "Categories": [
+        {
+            "name": "Our blog",
+            "quantity": "12",
+            "link": "#"
+        },
+        {
+            "name": "Metaverse",
+            "quantity": "24",
+            "link": "#"
+        },
+        {
+            "name": "Gaming",
+            "quantity": "24",
+            "link": "#"
+        },
+        {
+            "name": "Art",
+            "quantity": "24",
+            "link": "#"
+        },
+        {
+            "name": "Robot",
+            "quantity": "24",
+            "link": "#"
+        },
+        {
+            "name": "Launchpad",
+            "quantity": "24",
+            "link": "#"
+        },
+        {
+            "name": "Crypto",
+            "quantity": "430",
+            "link": "#"
+        }
+    ],
+    "Tags": [
+        {
+            "name": "NFT",
+            "link": "#"
+        },
+        {
+            "name": "ICO",
+            "link": "#"
+        },
+        {
+            "name": "Robot",
+            "link": "#"
+        },
+        {
+            "name": "Crypto",
+            "link": "#"
+        },
+        {
+            "name": "Launchpad",
+            "link": "#"
+        },
+        {
+            "name": "Metaverse",
+            "link": "#"
+        }
+    ],
+    "figure": {
+        "path": "img/images/blog-page/basilisk-main-2x.webp",
+        "desc": "A massive, winged basilisk-like creature in a misty teal environment, with a lone robed figure standing nearby",
+        "caption": "Photo caption"
+    },
+    "mainSection": {
+        "title": "The Basilisks NFT",
+        "button": {
+            "link": "#",
+            "label": "Gaming"
+        },
+        "writer": "Tony Nguyen",
+        "date": "20 Jun 2022",
+        "paragraphs": [
+            {
+                "text1": "The Basilisks that",
+                "span": "players",
+                "text2": "collect are represented with NFTs on Ethereum’s blockchain with real-world value. Players use the",
+                "highlight": "Basilisks they have",
+                "text3": "gathered to battle other players to win ether (ETH)."
+            },
+            {
+                "text": "In Balthazar Dragons, players are immersed in a 3D open world to explore and capture dragon-like beasts called Basilisks. Balthazar Dragons is an upcoming fantasy role-playing game developed on the Ethereum blockchain by a decentralized autonomous organization (DAO) called the Balthazar DAO."
+            }
+        ],
+        "list": [
+            "The Basilisks that players collect are represented with NFTs on Ethereum’s blockchain...",
+            "In Balthazar Dragons, players are immersed in a 3D open world... <span class=\"page__text-highlight\">game developed</span>..."
```

Comment:
Could you clarify please, why we have some tag here? Do we have another solution?

Reply:
That's right, you can do it sometimes. The main thing is that you have an understanding of why